client/my-sites/checkout/checkout-thank-you/redesign-v2/products/jetpack-search-product.tsx

```diff
@@ -0,0 +1,57 @@
+import { Button } from '@automattic/components';
+import { translate } from 'i18n-calypso';
+import ThankYouProduct from 'calypso/components/thank-you-v2/product';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { useSelector } from 'calypso/state';
+import {
+	getJetpackSearchCustomizeUrl,
+	getJetpackSearchDashboardUrl,
+} from 'calypso/state/sites/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+export type ThankYouJetpackSearchProductProps = {
+	purchase: ReceiptPurchase;
+	siteId: number | null;
+};
+
+export default function ThankYouJetpackSearchProduct( {
+	purchase,
+	siteId,
+}: ThankYouJetpackSearchProductProps ) {
+	const selectedSite = useSelector( getSelectedSite );
+	const jetpackSearchCustomizeUrl = useSelector( ( state ) =>
+		getJetpackSearchCustomizeUrl( state, siteId )
+	);
+	const jetpackSearchDashboardUrl = useSelector( ( state ) =>
+		// At this point in the flow, having purchased a product for a specific
+		// site, we can assume that `siteId` is a number and not `null`.
+		getJetpackSearchDashboardUrl( state, siteId as number )
+	);
+
+	const productButtonLabel = selectedSite?.jetpack
+		? translate( 'Go to Search Dashboard' )
+		: translate( 'Customize Search' );
+	const productButtonHref = selectedSite?.jetpack
+		? jetpackSearchDashboardUrl
+		: jetpackSearchCustomizeUrl;
+
+	const recordThankYouClick = () => {
+		recordTracksEvent( 'calypso_jetpack_product_thankyou', {
+			product_name: 'search',
+			value: 'Customizer',
+			site: 'wpcom',
+		} );
```

Comment:
@chad1008 I came to realize that there's probably a bug in the current tracking code, as we're reporting the same value regardless of which button we render ("Go to Search Dashboard" or "Customize Search"). Let's take the opportunity to fix tracking such that we report different values for a Jetpack site:
* `value: 'Search Dashboard'`
* `site: 'jetpack'`

Reply:
Done! 773fe93134

The two CTA flavors will now track separately. Glad you spotted that!